commit 4580cd7ec22873a70a16de694262661e2af0fef8
Author: Tracey Dent <tdent48227@gmail.com>
Date:   Thu Oct 7 20:01:42 2010 -0400

    Staging: rtl8712: Makefile: replace the use of <module>-objs with <module>-y
    
    Changed <module>-objs to <module>-y in Makefile.
    
    Signed-off-by: Tracey Dent <tdent48227@gmail.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/rtl8712/Makefile
+++ b/drivers/staging/rtl8712/Makefile
@@ -1,4 +1,4 @@
-r8712u-objs :=				\
+r8712u-y :=				\
 		rtl871x_cmd.o		\
 		rtl8712_cmd.o		\
 		rtl871x_security.o	\
